### DISCUSSION TITLE
Web clip login

### DISCUSSION THREAD

**Discussion by cveres** [codecentric/web-clip#34] [Q&A]
I managed to log in to a community solid server in the WebClip setup screen.
However, the WebClip popup asks me to login in again. It seems to know that I am logged in as it only asks to authorize the app. When I try I get the error in the screenshot.
![Screenshot 2023-04-29 19 24 05](https://user-images.githubusercontent.com/1199713/235295604-2f26ed48-6780-4f30-aeff-76b12f122353.png)

**Comment by angelo-v** [comment]
Sorry for the late response. Can you try to clear all caches and/or re-install webclip? If this does not help please give me the console output of the developer console of the browser window and as well the background script (If you need help how to do this let me know). Can you tell which version of CSS you tried? You can also try with an account on https://solidweb.me which should work.
